app/views/admin/books/_form.html.erb

```diff
@@ -61,6 +61,21 @@
 
   <%= render "admin/books/categories_form", form: form %>
 
+  <div class="row">
+    <div class="col-6">
+      <%= render "admin/books/form/position", form: form %>
+      <p class="form-text text-muted">This will set the order that
+      this book shows up on <a href="https://crimethinc.com/books">crimethinc.com/books</a>.</p>
+    </div>
+
+    <div class="col-6">
+      <%= form.check_box :hide_from_index %>
+      <%= form.label :has_index, "Hide this book from the index page?" %><br>
+      <p class="form-text text-muted">This will prevent the book from
+      showing up on <a href="https://crimethinc.com/books">crimethinc.com/books</a>.</p>
```

Comment:
```suggestion
      showing up on <a href="/books">crimethinc.com/books</a>.</p>
```

So that this link does The Right Thing™ for localized /admin users.